### DISCUSSION TITLE
Query syntax for using workflowRun and workflow object of GitHub GraphQL API?

### DISCUSSION THREAD

**Discussion by SaurabhRai19** [graphql/graphiql#2315] [Q&A / Support] [answered]
Hi. I am facing an issue with writing the query that can display info of the workflows running inside a repository. There exists [workflowRun and workflow](https://docs.github.com/en/enterprise-server@3.3/graphql/reference/objects#workflow) object for this requirement. But can’t find the right syntax to use these objects for returning the workflow-related info.

**Comment by timsuchanek** [comment, marked as answer]
Hi @SaurabhRai19 - this repository is about GraphiQL, the GraphQL IDE.
It seems like your question is about the GitHub GraphQL API.
I encourage you to have a look at https://github.com/github/feedback/discussions to give feedback there. Thanks!
